pages/pipelines/managing_priorities.md.erb

```diff
@@ -1,5 +1,7 @@
 # Job Prioritization
 
+Jobs dispatch is when the job is taken from the queue and assigned to an agent.
+
 Jobs are dispatched to agents in the following order
```

Comment:
What do you think about using this instead:

> Jobs are dispatched (taken from the queue and assigned to agents) in the following order

This way we're not leading a section on Prioritization with a note on dispatch